src/Controller/BannerController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use WMDE\BannerServer\UseCase\BannerProvideRequest;
+use WMDE\BannerServer\UseCase\BannerProvideUseCase;
+
+/**
+ * @license GNU GPL v2+
+ */
+class BannerController {
```

Comment:
Good that you don't use the Symfony-provided Controller that, that'll keep your from using the DI container for getting services